src/main/java/com/facebook/genAlgo/evaluator/Variant1EvaluatorImpl.java

```diff
@@ -2,16 +2,18 @@
 
 import com.facebook.genAlgo.gene.Gene;
 
-public class Variant1EvaluatorImpl implements Evaluator {
+import static java.lang.Math.log;
 
-    private final char target;
+public class Variant1EvaluatorImpl implements Evaluator {
 
-    public Variant1EvaluatorImpl(char target) {
-        this.target = target;
-    }
+  private final char target;
 
-    @Override
-    public float setFitness(Gene gene) {
-        return 0;
-    }
+  public Variant1EvaluatorImpl(char target) {
+    this.target = target;
+  }
+  @Override
+  public float setFitness(Gene gene) {
+    float delta = gene.getValues()[0] - target;
```

Comment:
this is not stated directly in docs, but we assume that delta is never below zero, so Math.abs() will solve this